### ISSUE TITLE
[Feature Request] Expose preburn balance in get_account_state JSON RPC view

### ISSUE THREAD

**Issue by ankushagarwal** [libra/libra#4930]
## Motivation

We do not expose preburn balance(s) in get_account_state JSON RPC method and hence have no way for DDs to query the  balance(s) in the preburn area

Here is the method which needs updating : https://github.com/libra/libra/blob/54ff18c801dd8fc3f639956b3e1bb36f93dd5ddc/json-rpc/src/methods.rs#L101. Once we obtain the `AccountState` blob, we need to parse it to obtain the preburn balance(s) associated with the account and expose them in the `AccountView` struct.

@moeziniaf : Assigning to you since I think you might have the best context on how to get this done.

**Comment by moeziniaf** [comment]
Yup, I'm modifying preburn right now to contain one balance, so this will be blocked on that since Move prover changes need to be made before we can land that.
